basalt_runner/CMakeLists.txt

```diff
@@ -0,0 +1,285 @@
+cmake_minimum_required(VERSION 3.10)
```

Comment:
It would be good to include an own README file in `basalt_runner`, which just says that the some files in that folder are from Basalt and licensed under BSD (the license text is in the affected files so it's not necessary to have a sepearete LICENSE file IMO)